### PR TITLE
[app platform] Add dashboard context menu feature

### PR DESCRIPTION
**Pull request by DrkSephy**
This pull request adds a new feature to the SDK, `dashboard_context_menu`.

### CODE REVIEW

File: src/features/index.ts [hunk 1]
```diff
@@ -8,6 +8,16 @@ import { FeatureManager } from './featureManager';
 
 export type { FeatureManager } from './featureManager';
 
+class DashboardContextMenuManager extends FeatureManager {
+    type = UiAppFeatureType.DASHBOARD_CONTEXT_MENU;
+    eventsToSubscribe = [UiAppEventToSubscribeType.DASHBOARD_CONTEXT_MENU];
```
Comment (nmuldavin) [inline src/features/index.ts:13]: What does this event do?

Comment (DrkSephy) [reply src/features/index.ts:13]: Before the recent changes, it would inform the 3p app when the context menu is triggered -- this may be outdated now.

Comment (DrkSephy) [reply src/features/index.ts:13]: Another note: this event doesn't send any new information to the 3p app as it is still TBD what type of context would be useful.

Comment (nmuldavin) [reply src/features/index.ts:13]: thanks for the context!

**Review by enbashi — APPROVED** [review]

**Pull request merged**